modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1044,11 +1073,43 @@ Use a different runtime or remove the runtime option to run the query with the d
 ----
 SHOW INDEXES YIELD *
 ----
-====
+
+======
+[.include-with-GQLSTATUS-code]
+======
+Query::
++
+[source,cypher]
+----
+CYPHER runtime=pipelined SHOW INDEXES YIELD *
+----
+
+Returned GQLSTATUS code::
+01N40
+
+Returned Status Description::
+warn: runtime unsupported.
+The query cannot be executed with `runtime=pipelined` is used.
+Cause: `PIPELINED`.
+
+Suggestions for improvement::
+Use a different runtime or remove the runtime option to run the query with the default runtime:
++
+[source,cypher]
+----
+SHOW INDEXES YIELD *
+----
+======
+=====
 
 [#_neo_clientnotification_statement_runtimeexperimental]
 === RuntimeExperimental
 
+[WARN]
```

Comment:
I don't know whether it is a correct decision to add a warning here. But there is no GQL status for this notification. At the same time it's in use in versions prior 5.14, so we can't just remove it